internal/authflow/flow.go

```diff
@@ -72,7 +72,7 @@ func AuthFlow(oauthHost string, IO *iostreams.IOStreams, notice string, addition
 				return nil
 			}
 
-			fmt.Fprintf(w, "%s to open %s in your browser... ", cs.Bold("Press Enter"), oauthHost)
+			fmt.Fprintf(w, "%s to open %s in your browser... ", cs.Bold("Press Enter"), authURL)
```

Comment:
This will probably work to print the URL, but we don't want to print the full URL if our tool is also going to attempt to open the URL ourselves, since that could be confusing to the user (should they open the printed URL, or press Enter? some will do both). Instead, per my comment in the original issue https://github.com/cli/cli/issues/5500#issuecomment-1117504829, if the user intends to open the URL manually themselves, they should indicate so in the `gh auth login` interactive prompt, which should then print the URL and _not_ offer the functionality to press Enter to have gh open it.

Reply:
I disagree, and I'm not planning to work on your suggestion for that reason. Even if people are confused by this, doing it twice is still fine. Compared to that, asking people another multi-choice question is worse and adds unnecessary complexity. The third option of not changing it at all means that some use cases are dropped on the floor, and the affected users can't login and have to find a workaround to use your CLI at all.

Reply:
For precedence, see `aws sso login` or `gcloud auth login`. Btw. both of those work for me out of the box.

Reply:
@mislav Asking the user to specify they will open the URL manually is poor UX. Just let the user decide. Tell them you'll open the browser for them, but just in case that doesn't work, here is the URL.

End of story.

If people can figure out how to use `git` then they can figure out what to do with a link on a screen.